src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -11,16 +10,77 @@
 
 import org.mindrot.jbcrypt.BCrypt;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import codeu.model.data.User;
+import codeu.model.data.Activity;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import java.util.*;
 
+/** Servlet class responsible for the Activity Feed page. */
 public class ActivityFeedServlet extends HttpServlet {
-  
+	
+	/** Store class that gives access to Users. */
+	private UserStore userStore;
+	  
+	/** Store class that gives access to Conversations. */
+	private ConversationStore conversationStore;
+
+	/** Store class that gives access to Messages. */
+	private MessageStore messageStore;
+
+	 /**
+	  * Set up state for handling login-related requests. This method is only called when running in a
+	  * server, not when running in a test.
+	 */
+	@Override
+	public void init() throws ServletException {
+       super.init();
+       setConversationStore(ConversationStore.getInstance());
+       setMessageStore(MessageStore.getInstance());
```

Comment:
small nit: as per the style guide, these lines should be indented like line 44!

Reply:
Done.